code/modules/mining/equipment/regenerative_core.dm

```diff
@@ -42,6 +42,8 @@
 		go_inert()
 
 /obj/item/organ/regenerative_core/proc/preserved(implanted = 0)
+	if(inert)
+		return
 	inert = FALSE
```

Comment:
Unecessary since you are already checking this is the case and early returning in the line above